monai/data/synthetic.py

```diff
@@ -14,7 +14,7 @@
 from monai.transforms.utils import rescale_array
 
 
-def create_test_image_2d(width, height, num_objs=12, rad_max=30, noise_max=0.0, num_seg_classes=5):
+def create_test_image_2d(width, height, num_objs=12, rad_max=30, noise_max=0.0, num_seg_classes=5, channel_dim=None):
     """
     Return a noisy 2D image with `numObj' circles and a 2D mask image. The maximum radius of the circles is given as
     `radMax'. The mask will have `numSegClasses' number of classes for segmentations labeled sequentially from 1, plus a
```

Comment:
Add 'channel_dim' explanation to docstring.

Reply:
Updated in the latest patch.
Thanks.